packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -741,6 +741,23 @@ class Auth {
         throw error
     }
 
+    /**
+     * This method extracts the custom parameters from a set of parameters.
+     *
+     * Custom parameters are defined as those whose keys are prefixed with 'c_'
+     * @Internal
+     */
+    extractCustomParameters = (
```

Comment:
It doesn't look like this is specific for auth. How do you feel if we move this out from the auth class and move it into a utils function?

Reply:
Good point. This works for any SCAPI call where the custom parameters are mixed in with the regular params.

I'll move this to utils